README.md

```diff
@@ -54,6 +54,113 @@ The following Visual Studio Code extensions are recommended:
 - [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
 - [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
 
+## Scripts and tasks
+
+Scripts for each `package.json` should follow these rules:
+
+### Nesting
+
+Some scripts may be nested under other scripts using the colon (`:`) symbol. This can interchangeably be used to mean either:
```

Comment:
A flaw with this approach is that Nx's CLI uses the colon symbol to divide the project name, target name and configuration name (see [Use Executor Configurations](https://nx.dev/recipes/executors/use-executor-configurations), which uses `nx run mylib:build:production` as an example).

The only impact I noticed from this is that, when running a script whose name contains a colon using Nx, we can't use the short syntax, we have to use `--target=`.

Reply:
I don't think the above problem is necessarily bad enough on its own to justify changing the symbol. However, if we were to replace scripts with [`nx:run-command` executors](https://nx.dev/packages/nx/executors/run-commands#nx:run-commands) and [executor configurations](https://nx.dev/recipes/executors/use-executor-configurations), this problem would be inherently solved and we could keep only the relevant scripts in `package.json` files.

Reply:
I found another problem with using the `:` symbol. When I run targets from vscode using the extension, I get errors.